build/dev-server.js

```diff
@@ -1,4 +1,4 @@
-require('./check-versions')()
+// require('./check-versions')()
```

Comment:
Why did you comment this out?

Reply:
comman yarn start was hanging at this point. This comment fixed it. Don't understand why :(